.gocd/build.gocd.groovy

```diff
@@ -511,18 +511,6 @@ GoCD.script {
           pipeline = 'code-sign'
           stage = 'metadata'
         }
-        dependency('upload-addons') {
-          pipeline = 'upload-addons'
-          stage = 'upload-addons'
-        }
-        dependency('go-packages') {
-          pipeline = 'go-packages'
-          stage = 'fetch_from_build_go_cd'
-        }
-        dependency('go-addon-build') {
```

Comment:
We need to remove `enterprise` material as well right?

Reply:
Yes, sorry missed that.

Reply:
It needs to be updated to `gocd` repo right?

Reply:
I don't see a need to add `gocd` as material. Let's talk later.